engine/src/test/java/org/hibernate/validator/test/internal/xml/Parent.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.test.internal.xml;
+
+/**
+ * Test class for HV-1534.
+ *
+ * @author robd
+ */
+public class Parent {
+
+	String parentAttribute = null;
+
+	Parent( String parentAttribute ) {
+		this.parentAttribute = parentAttribute;
+	}
+
+	public final String getParentAttribute() {
```

Comment:
Let's remove the final.

Reply:
If you're happy, I'd prefer this to remain final as it reflects the specific issue we have in our codebase that would be part of the problem we'd face if this fix was deemed unacceptable.

Reply:
OK, makes sense.